tests/figures/chart/charts_component.test.ts

```diff
@@ -2421,9 +2642,10 @@ test("ChartJS charts extensions are loaded when mounting a spreadsheet, are only
   const spyUnregister = jest.spyOn(window.Chart, "unregister");
   createChart(model, { type: "bar" }, chartId);
   await mountSpreadsheet();
-  expect(spyRegister).toHaveBeenCalledTimes(7);
+  expect(spyRegister).toHaveBeenCalledTimes(8);
```

Comment:
Is it possible to change the `7` to `regiustry.getKeys().length` or something dynamic instead ? It's a bit annoying having to update the 2 tests each time 🙈